packages/parser/lib/css_parser.mly

```diff
@@ -89,29 +89,131 @@ bracket_block (X): xs = delimited(LEFT_BRACKET, X, RIGHT_BRACKET) { xs }
 paren_block (X): xs = delimited(LEFT_PAREN, X, RIGHT_PAREN) { xs }
 
 /* https://www.w3.org/TR/mediaqueries-5 */
-/* Parsing with this approach is almost as good the entire spec */
 
-prelude: xs = loption(nonempty_list(loc(value_in_prelude))) { xs }
-
-/* Missing grammars: */
-/* (width >= 600px) */
-/* (400px < width < 1000px) */
-/* (not (color)) and (not (hover)) */
-/* Combinator "," */
-media_query_prelude_item:
-  | i = IDENT { Ident i }
+// TODO: Alot of duplication here, any way to improve it?
+/* ">" */
+greater_operator: o = OPERATOR { o }
+/* "<" */
+less_operator: o = OPERATOR { o }
+/* "=" */
+equal_sign: e = OPERATOR { e }
+/* "and" */
+and_operator: o = OPERATOR { o }
+/* "or" */
+or_operator: o = OPERATOR { o }
+/* "only" */
+only_operator: o = OPERATOR { o }
+/* "not" */
+not: n = OPERATOR { n }
+/* "all" | "screen" | "print" */
+media_type: s = IDENT { s }
```

Comment:
Maybe have a token for each of the media_types? Feel liks ident would make the parsing more complex here.

Reply:
That makes sense, I think we need to do the same for the logical operators, `and` & `or`